python/hail/dataset.py

```diff
@@ -1285,56 +1285,82 @@ def concordance(self, right):
 
         **Example**
 
-        >>> concordance_pair = vds.concordance(hc.read('data/example2.vds'))
+        >>> summary, samples, variants = vds.concordance(hc.read('data/example2.vds'))
 
         **Notes**
 
-        The `concordance` command computes the genotype call concordance between two bialellic variant datasets. The concordance
-        results are stored in a global annotation of type Array[Array[Long]], which is a 5x5 table of counts with the
-        following mapping:
+        The `concordance` command computes the genotype call concordance between two bialellic variant datasets. 
+        It performs an inner join on samples (only samples in both datasets will be considered), and an outer join
+        on variants. This method returns a tuple of three objects: a nested list of list of int with global concordance
```

Comment:
It might be helpful to be a bit more verbose about what happens to a variant that is only in one dataset (people won't understand what outer join means).